lib/validators/options_processor/options/no_empty.js

```diff
@@ -26,9 +26,14 @@ class NoEmpty {
     if (typeof input === 'string') {
       isMergeable = input.trim().length !== 0
     } else if (Array.isArray(input)) {
-      isMergeable = input.length !== 0
+      isMergeable = true
     } else {
-      throw new Error(UNKNOWN_INPUT_TYPE_ERROR)
+      if (input == null) {
+        // Fallback if `input` is `null` or `undefined`
+        isMergeable = true
```

Comment:
if no empty is `enabled`, why should this be true? shouldn't null or undefined be considered `false`?

Reply:
@shine2lay thanks for the tip!